docs/getting-started/browser-not-supported/index.hbs

```diff
@@ -0,0 +1,85 @@
+---
+title: Browser not supported page
+description: asdf
+nav: side/getting-started/browser-not-supported
+order: 3
+lunr: true
+toc:
+  - title: Redirect
+    slug: redirect
+  - title: Overlay
+    slug: overlay
+---
+{{#> page}}
+  {{#*inline "content"}}
+    <h1 class="docs-h2">{{title}}</h1>
+
+    <p class="lead">
+      You're probably not going to support every browser in the world
+      (Yes, we're talking about you Internet Explorer.)
+      We recommend showing a "Browser not supported" page instead of
+      displaying a page with rendering errors and broken behavior.
+    </p>
+
+    <p>
+      There are two good ways to show users of old IE versions (version <= 9).
+      Both are based on IE's conditional comments features.
+    </p>
+    
+    {{> table-of-contents data=toc}}
+
+    <h2 id="redirect" class="docs-h3">Redirect</h2>
+    <p>
+      You can redirect to another URL that shows the user the
+      "Browser not supported" page using a meta tag.
+    </p>
+
+    {{#code-snippet "html"}}
+      <!DOCTYPE html>
+      <html>
+        <head>
+          <!--[if lte IE 9]><meta http-equiv="refresh" content="0; URL=/your/browser-not-supported-page.html"><![endif]-->
+          Meta Tags here.
```

Comment:
Just for copy-pasta's sake, I'd make this a comment. Also it gets highlighted in a more subtle way if it's a comment, showing users that it's only informational.

```
<!-- Meta Tags here. -->
```